.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,28 +1,35 @@
 Fixes #
 
-(let GitHub automatically close an issue when this pull request gets merged)
+<!-- (let GitHub automatically close an issue when this pull request gets merged) -->
 
+<!--
 ## Before you submit a pull request
+
 This template is not necessary when you do simple things like updating packages to the latest version. But in doubt, it's always better so provide some information.
+-->
 
 ## Description
+<!--
 Provide a description, what your changes do and why they are important
 
 Please link issues and other pull requests connected to this one.
+-->
 
 ## Additional information
-Mention things we might need to know. Like:
+<!-- Mention things we might need to know. Like: -->
 
 Works properly:
 - [x] x86_64
-- [ ] aarch64 (reasons why it doesn't)
+- [ ] i686
+- [ ] aarch64 <!-- (reasons why it doesn't) -->
```

Comment:
Would it make sense to just change this to `- [  ] arm` for now?
There are **ZERO** Chromebooks which use an aarch64 userspace at the moment, and apparently, that won't even start happening until later this year. So there is really no reason to go that granular at this point.

Reply:
Thoughts @uberhacker ?

Reply:
I could get onboard with `- [ ] armv7l`.

Reply:
Change made